scripts/install.sh

```diff
@@ -0,0 +1,308 @@
+#!/bin/bash
+
+#########################
+# Repo specific content #
+#########################
+
+VERIFY_CHECKSUM=${VERIFY_CHECKSUM:-'0'}
+ALIAS_NAME=${ALIAS_NAME:-''}
+OWNER=${OWNER:-'cnrancher'}
+REPO=${REPO:-'autok3s'}
+SUCCESS_CMD="$REPO version"
+BINLOCATION=${BINLOCATION:-'/usr/local/bin'}
+KUBEEXPLORER_REPO=${KUBEEXPLORER_REPO:-'kube-explorer'}
+KUBEEXPLORER_DOWNLOAD_URL=https://github.com/$OWNER/$KUBEEXPLORER_REPO/releases/download
+KUBEEXPLORER_VERSION=$(curl -sI https://github.com/$OWNER/$KUBEEXPLORER_REPO/releases/latest | grep -i "location:" | awk -F"/" '{ printf "%s", $NF }' | tr -d '\r')
+
+#   - INSTALL_AUTOK3S_MIRROR
+#     For Chinese users, set INSTALL_AUTOK3S_MIRROR=cn to use the mirror address to accelerate
+#     autok3s binary file download, and the default mirror address is rancher-mirror.cnrancher.com
```

Comment:
Should replace "cnrancher.com" with "rancher.cn", "cnrancher.com" may be deprecated in the future